vent/io/devices/__init__.py

```diff
@@ -0,0 +1,439 @@
+"""
+Subclass implementations of specific sensor devices.
+"""
+from abc import ABC
+from collections import OrderedDict
+
+import pigpio
+
+
+class IODeviceBase(ABC):

```

Comment:
See my comment on the other __init__.py. I think it's best to just list what is exported by a module in the __init__ files and have the actual code in other .py files.

Reply:
Also done. See commit [eea563c](https://github.com/CohenLabPrinceton/Ventilator-Dev/pull/45/commits/eea563c22a289e66ec7b671626bdc451578c78b5) in PR #45